cmd/cluster-capacity/app/options/options.go

```diff
@@ -71,7 +72,7 @@ func NewClusterCapacityOptions() *ClusterCapacityOptions {
 
 func (s *ClusterCapacityOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", s.Kubeconfig, "Path to the kubeconfig file to use for the analysis.")
-	fs.StringVar(&s.PodSpecFile, "podspec", s.PodSpecFile, "Path to JSON or YAML file containing pod definition.")
+	fs.StringVar(&s.PodSpecFile, "podspec", s.PodSpecFile, "Path to JSON or YAML file containing pods definition.")
```

Comment:
Ok nvm, I just checked, and multiple pod specs in a json files works. But as I said, it would be good to better document it.